### DISCUSSION TITLE
Incorrect Directory Name in GitHub Desktop

### DISCUSSION THREAD

**Discussion by Wlevy1** [community/community#64541] [Repositories]
### Repositories

Bug

### Body

I'm new in Github Desktop I have only 2 weeks of use and I didn't have any issues until now, I'm tryna open my repository and it shows "_Can't find HTML-CSS_"(repository name)
![image](https://github.com/community/community/assets/136914338/c629587d-7576-4c04-b93f-15d7eeafa30f)
I realized that the problem is that its reading "Documents" instead of "Documentos" because english is not my main language I speak portuguese and rarely use english in my pc

Obs: I tried to place "HTML-CSS" in my desktop so I can locate it from there but it reads "C:\Users\welli\Desktop\HTML-CSS" so still doesn't work since "desktop" in portuguese is "área de trabalho"

I tried to reinstall Github Desktop but it didn't work, is there anything I can do?

**Comment by ghost** [comment]
Pretty sure you can just get the file opened through locate… button. Or just open a new window and put the new workflow in.

**Reply by Wlevy1** [reply]
I've tried but nothing changes, but for some reason when I cloned it wrote Documentos instead os Documents so.. ye thank you anyway!

**Comment by steveward** [comment]
@Wlevy1 sorry for the trouble! Can you try removing the repository from GitHub Desktop by clicking `Remove`, and then go to the directory in File Explorer on Windows and try dragging-and-dropping the repository directly into GitHub Desktop.

If that doesn't work, it would be helpful if you could share the log file so we could see if there is a specific issue. To access the log files go to the file menu in GitHub Desktop and select `Help` > `Show Logs`. The log files are created daily -- please upload a log file as an attachment from a day where you experienced the issue.

**Reply by Wlevy1** [reply]
Sure!
